src/platyPS/platyPS.psm1

```diff
@@ -433,6 +434,10 @@ function Update-MarkdownHelp
                     return
                 }
             }
+            elseif ($loadedModulesBefore.Name -notcontains $command.ModuleName)
```

Comment:
can we also handle a case, when `$command.Module` is empty? It needs a separate treatment, so we don't have this warning for the inline defined functions.

Example:
![image](https://user-images.githubusercontent.com/816680/51366939-2d143f00-1a9d-11e9-8d4e-39ec22dbf081.png)

Reply:
Someghint like 
```
elseif (($loadedModulesBefore.Name -notcontains $command.ModuleName) -and ($null -ne $command.ModuleName))
```
?

I'm not sure what exactly is the case.

Reply:
yes, pretty much. I'd swap the conditions to make $null check first.

Reply:
Done.